examples/contact_form/src/main.rs

```diff
@@ -0,0 +1,39 @@
+#[macro_use] extern crate nickel;
+use nickel::{Nickel, HttpRouter};
+use std::collections::HashMap;
+
+// Required For Post
+extern crate hyper;
+use std::io::Read;
+// Required For Post
+
+fn main() {
+
+    //Implement A Server
+    let mut server = Nickel::new();
```

Comment:
nit: weird spacing after this line, also prefer not to have blank lines after opening braces (after main, and within each `middleware!`)